src/main/java/de/bonndan/nivio/output/Color.java

```diff
@@ -38,10 +37,10 @@ public static String nameToRGB(String name, String defaultColor) {
 
     public static String lighten(String color) {
         try {
-            java.awt.Color col = WebColors.getRGBColor("#" + color).brighter();
-            return Integer.toHexString(col.getRGB());
+            java.awt.Color col = java.awt.Color.decode(color.startsWith("#") ? color : "#" + color);
```

Comment:
Maybe you could choose a different name for your `Color` class... I don't know about best practice here, but I personally don't like using fully qualified names (`java.awt.Color`), if it can be avoided.

Reply:
Afaik there is no way around fully qualified names when the class has the same name.

Reply:
That iss correct. That's why I was asking if you'd change the name. I guess this could be a matter of taste.